init.py

```diff
@@ -109,3 +128,7 @@ def add_to_synced(fork_name: str):
         msg = 'Unreversable decision. Are you sure you want to remove all the content, all branches and releases from this repository?'
         if input(f"{msg} (y/N)? ").lower() == 'y':
             purge_content(man)
+
+    invite_ci_bot(man)
+    time.sleep(2)  # give github a while to process
```

Comment:
since we know we sent an invitation shouldnt we just wait untill it appears (with a timeout) instead of an arbitrary sleep?